go.mod

```diff
@@ -1,3 +1,10 @@
-module github.com/elastic/elastic-agent-changelog-tool
+module github.com/elastic/elastic-agent-changelog
```

Comment:
May you help me understand why this change has been introduced? Given how imports work in go this is very confusing.

If we want a nicer name I would prefer renaming the repository.

Reply:
This was originally done due to the way of how installing go packages work in the context of running a cli. Commands will take the module name by default, so elastic-agent-changelog [command] was the specs one, -tool just didn't respect the command format needed.

Reply:
:+1: I'll check with him on this

Reply:
Let's rename the module to `github.com/elastic/elastic-agent-changelog-tool`